iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateBranchFeeders.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.modification.topology;
+
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.extensions.ConnectablePosition;
+import com.powsybl.iidm.network.extensions.ConnectablePositionAdder;
+
+import java.util.Objects;
+
+/**
+ * This class allows to add a new branch's feeders on existing busbar sections. The voltage level containing the
+ * busbar section should be described in node/breaker topology. The branch is created and connected to
+ * the busbar sections with a breaker and a closed disconnector each. The branch is also connected to all
+ * the parallel busbar sections, if any, with an open disconnector.
+ *
+ * @author Miora Vedelago <miora.ralambotiana at rte-france.com>
+ */
+public class CreateBranchFeeders extends AbstractCreateConnectableFeeders {
```

Comment:
As we are creating bays also on each side, shouldn't the keyword `Bay` be in class name?